tests/test_wave_05.py

```diff
@@ -119,39 +146,48 @@ def test_delete_goal(client, one_goal):
     assert response_body == {
         "details": 'Goal 1 "Build a habit of going outside daily" successfully deleted'
     }
-
+    assert Goal.query.get(1) == None
     # Check that the goal was deleted
-    response = client.get("/goals/1")
-    assert response.status_code == 404
+    
 
-    raise Exception("Complete test with assertion about response body")
+    # raise Exception("Complete test with assertion about response body")
     # *****************************************************************
     # **Complete test with assertion about response body***************
     # *****************************************************************
 
 
-@pytest.mark.skip(reason="test to be completed by student")
+# @pytest.mark.skip(reason="test to be completed by student")
 def test_delete_goal_not_found(client):
-    raise Exception("Complete test")
+    # raise Exception("Complete test")
 
     # Act
+    response = client.delete("/goals/1")
+    response_body = response.get_json()
     # ---- Complete Act Here ----
 
     # Assert
+    assert response.status_code == 404
     # ---- Complete Assertions Here ----
     # assertion 1 goes here
+    assert response_body == {
+        "message": "Goal 1 not found"
+    }
+    
     # assertion 2 goes here
+    assert Goal.query.all() == []
     # ---- Complete Assertions Here ----
 
 
-@pytest.mark.skip(reason="No way to test this feature yet")
+# @pytest.mark.skip(reason="No way to test this feature yet")
 def test_create_goal_missing_title(client):
     # Act
     response = client.post("/goals", json={})
     response_body = response.get_json()
 
     # Assert
     assert response.status_code == 400
+    assert "details" in response_body
```

Comment:
I'm not sure this `assert` statement covers anything that isn't also covered by the `assert` that begins on the line below, line 190. That `assert` will also verify that `"details"` is in `response_body`.